electricitylci/coal_upstream.py

```diff
@@ -2,8 +2,178 @@
 # -*- coding: utf-8 -*-
 import pandas as pd
 from electricitylci.globals import data_dir, output_dir
+from electricitylci.eia923_generation import eia923_download
+import os
+from os.path import join
+from electricitylci.utils import find_file_in_folder
+import requests
+import PhysicalQuantities as pq
```

Comment:
Well, I think you've seen the extent to which pq is used so far. I just reworked someone else's fork the other day to get it to work with Python 3. I'm sure we could probably get away with 3-4 constants to handle most of the conversions we have. I just like to have non-rounded conversions present whenever I can. Anyhow, if PhysicalQuantities.py is present in the module, then no such need right? actually, I guess then it should be:

import electricitylci.PhysicalQuantities as pq.

I added in pq.py so that we could avoid having to add another package (such as pint).

Reply:
Ok, that works.